content/writing_tests/_index.md

```diff
@@ -38,7 +38,7 @@ Back then, the most common way to handle asynchronous code was via callbacks. Th
 
 You might have noticed that if a testcase in Sakuli v2 is wrapped within `(async () => /*...*/)().then(done)` this construct it is called *Immediately Invoked Function Expression* (IIFE). Basically, it is the definition of a function that is immediately invoked. This pattern is widely used in JavaScript to preserve scopes and namespaces, since every symbol, which is defined at the top level of a script, becomes part of the global namespace and may potentially collide with other scripts. In the era of <a href="https://medium.com/fusebox/beginner-web-developers-use-a-bundler-31ab0c91d2f5" target="_blank">web bundler</a> and <a href="https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/" target="_blank">ES-modules</a> this use-case became more and more irrelevant. IIFE on the other hand still remains important after the introduction of async/await in JavaScript. One caveat of this syntax is, that the `await` keyword is only available within an asynchronous function - because the engine will wrap the whole body of this function within a Promise. That means, that in order to be able to make use of the `await` keyword, your code has to be executed within an asynchronous function. On the top level IIFE, we declare the actual function as `async` and can make use of `await`. Since the result of this function is automatically turned into a Promise, we can invoke `then` and pass the `done` function to it. `done` is a Sakuli specific function, which indicates to Sakuli that the testcase script is finished. There is currently a discussion on the <a href="https://github.com/tc39/proposal-top-level-await" target="_blank">JavaScript proposal in stage 3</a>, which would introduce a *top level await* in JavaScript.
 
-If you want to avoid this construct completely, you can use the `then` function of a Promise. The example code of the <a href="/Docs/getting-started" target="_blank">Getting started</a> guide would look like this then:
+If you want to avoid this construct completely, you can use the `then` function of a Promise. The example code of the <a href="/docs/getting-started/" target="_blank">Getting started</a> guide would look like this then:
```

Comment:
```suggestion
If you want to avoid this construct completely, you can use the `then` function of a Promise. The example code of the <a href="/docs/getting-started/" target="_blank" rel="noopener">Getting started</a> guide would look like this then:
```

https://www.christoph-freyer.at/blog/links-absichern-mit-rel-noopener/#.Xg22MxdKiL4